src/nextjs-front/pages/users/[id].tsx

```diff
@@ -221,6 +198,31 @@ const UserProfilePage: NextPageWithLayout = ({}) => {
     });
   }
 
+  /* Send friendship invite */
+  const requestFriend = async (id: string, username: string) => {
+    const reqSent = await fetch (`/api/users/${getUserData().id}`, {
```

Comment:
You have to add the requestSent too to the sender. This field is used to check for friends page. It is better to add this here too not to "break" something I think.